airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobCreator.java

```diff
@@ -109,15 +126,23 @@ public long createGetSpecJob(String integrationImage) throws IOException {
                                       String destinationDockerImageName,
                                       List<StandardSyncOperation> standardSyncOperations)
       throws IOException {
+    final JsonNode sourceConfiguration = oAuthConfigSupplier.injectSourceOAuthParameters(
```

Comment:
@jrhizor is this the right place to be injecting the oauth secrets i.e: enqueue them directly in the jobs DB?

Reply:
It's probably fine for now but it will change in the near future. This is what I'm working on eliminating this week, still working out the details of what it will look like.

Most likely we will push the combined version to config sent to Temporal.